package.json

```diff
@@ -41,7 +41,8 @@
     "express": "^4.17.1",
     "nunjucks": "^3.2.0",
     "path": "^0.12.7",
-    "testcafe": "^1.6.1"
+    "testcafe": "^1.6.1",
+    "testcafe-reporter-nunit": "^0.1.2"
```

Comment:
we were just discussing this actually and saying testcafe should be a devDependencies instead. Would moving that cause you any problems?

Reply:
No idea, let's move it and find out. We can either merge this in and test it out after it's been moved; or we can put this on hold, move it, and i'll pull that change into this branch to test

Reply:
ok let's get this merged in 👍